kodilla-basic-tests/src/main/java/com/kodilla/abstracts/homework1/Shape.java

```diff
@@ -0,0 +1,36 @@
+package com.kodilla.abstracts.homework1;
+
+public abstract class Shape {
+
+    private double a;
```

Comment:
Klasa abstrakcyjna to czesc wspolna - nie powinno tu byc zatem pol, bo nie sa wspolne